include/CloudFileMember.h

```diff
@@ -47,27 +47,30 @@ class CloudFileMember
     enum class Type { Member, Owner };
 
     CloudFileMember();
-    CloudFileMember(const ContactHandler &contact, const Type type);
+    CloudFileMember(UserId memberId, Type type, ContactHandler contact = {});
```

Comment:
Let's use 2 constructors:
`CloudFileMember(UserId memberId, Type type)`
`CloudFileMember(ContactHandler contact, Type type)`

Reply:
What the benefits?